source/scpd/scp/SCPDriver.d

```diff
@@ -109,8 +128,17 @@ nothrow:
     // `computeTimeout` computes a timeout given a round number
     // it should be sufficiently large such that nodes in a
     // quorum can exchange 4 messages
-    milliseconds computeTimeout(uint32_t roundNumber);  // Slot in the vtable
-
+    version (Windows)
+    {
+        // TODO Temporary action due to MSVC muggling problem
```

Comment:
mangling, not muggling.
Also are you sure it's 20700 ? It *might be related* to it.